### PR TITLE
Updated some instances of glatos, lesson title, deleted/renamed some …

### PR DESCRIPTION
**Pull request by jackVanish**
Changed some instances of GLATOS to 'glatos' to refer to the package instead of the network, updated some title text, and deleted/renamed some lessons. Didn't touch anything in the code since we're going to have to rework all of that anyway once we get the data.

### CODE REVIEW

File: _episodes/05-r-telemetry-report-array.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ questions:
 
 ### Mapping GLATOS stations - Static map
 
-This section will use the Receivers CSV for the entire GLATOS Network.
+This section will use the Receivers CSV for the entire ACT Network.
```
Comment (jdpye) [inline _episodes/05-r-telemetry-report-array.md:12]: It's going to have a subset of ACT stations that are cleared to go public. We should also change the header from Mapping GLATOS stations - static map

Comment (naomitress) [inline _episodes/05-r-telemetry-report-array.md:12]: this is not correct, we are using a subset based on approved projects for public consumption, consider rewording

File: _episodes/07-introduction-to-glatos.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ mark, followed by the name of the function.
 
 With our file path in hand, we'll want to use the read_otn_detections function
 to load our data into a dataframe. In this case, our data is formatted in the FACT
```
Comment (jdpye) [inline _episodes/07-introduction-to-glatos.md:56]: we can say OTN style here.

File: _episodes/07-introduction-to-glatos.md [hunk 3]
```diff
@@ -177,7 +177,7 @@ more amenable to plotting by reducing it from detections to detection events.
 Detection Events differ from detections in that they condense a lot of temporally and
 spatially clustered detections for a single animal into a single detection event. This is
 a powerful and useful way to clean up the data, and makes it easier to present and
-clearer to read. Fortunately, GLATOS lets us to this easily.
+clearer to read. Fortunately, glatos lets us to this easily.
```
Comment (jdpye) [inline _episodes/07-introduction-to-glatos.md:180]: *do

**Review by jdpye — CHANGES_REQUESTED** [review]
Couple small fixes, and a merge conflict to resolve.

**Review by naomitress — APPROVED** [review]
just the one comment regarding the receiver records we are bringing in, otherwise g2g!

File: _episodes/05-r-telemetry-report-array.md (hunk 1 above)
Comment (naomitress) [reply _episodes/05-r-telemetry-report-array.md]: changes made

File: _episodes/07-introduction-to-glatos.md (hunk 2 above)
Comment (naomitress) [reply _episodes/07-introduction-to-glatos.md]: changes made

File: _episodes/05-r-telemetry-report-array.md (hunk 1 above)
Comment (naomitress) [reply _episodes/05-r-telemetry-report-array.md]: duplicate of below

**Review by jdpye — APPROVED** [review]

**Pull request merged**